src/test/java/eu/dissco/backend/repository/ElasticSearchRepositoryIT.java

```diff
@@ -91,83 +85,95 @@ static void initContainer() {
     client = new ElasticsearchClient(transport);
   }
 
-  @BeforeEach
-  void initRepository() {
-    repository = new ElasticSearchRepository(client, formatter);
+  @AfterAll
+  public static void closeResources() throws Exception {
+    restClient.close();
   }
 
-  @AfterEach
-  void clearIndex() throws IOException {
-    client.indices().delete(b -> b.index(INDEX));
+  private static Stream<Arguments> provideKeyValue() {
+    return Stream.of(
+        Arguments.of("digitalSpecimen.ods:physicalSpecimenId.keyword", "global_id_45634"),
+        Arguments.of("q", PREFIX + "/0")
+    );
   }
 
-  @AfterAll
-  public static void closeResources() throws Exception {
-    restClient.close();
+  @BeforeEach
```

Comment:
Is there a reason you reordered these? I think @BeforeAll, @BeforeEach, @AfterAll is clearer.